build.sbt

```diff
@@ -33,13 +34,16 @@ lazy val scala211   = "2.11.12"
 lazy val scala212   = "2.12.6"
 lazy val scalaJs213 = "2.13.0-M4" // Scala.js does no have -pre
 
-lazy val scala213 = "2.13.0-M4"
-// lazy val scala213 = "2.13.0-pre-3ae6282" // use the sbt command `latest-213` to fetch the latest version
+lazy val scala213    = "2.13.0-M4"
+lazy val scala213Pre = LatestScala.getLatestScala213()
```

Comment:
This is a both a blessing and a curse. It's good since it can allow us to catch bugs early, It's bad since it's makes things more fragile.